openapi/v1/flowCytometryUser.yaml

```diff
@@ -2,6 +2,7 @@ openapi: 3.1.0
 info:
   description: |-
     This swagger page describes the flowCytometryUser API endpoints for ODM. These are typically used to find and retrieve flow cytometry data and metadata.
```

Comment:
Minor: while we're here, maybe it's a good time to unify this text for both flowCytometryCurator/flowCytometryUser  pages? There are some slight differences in them

Reply:
I didn't feel enough courage to do that, but now I feel. Will do after merging develop with Nikola's changes

Reply:
done

Reply:
also decided to remove read the docs